tests/test_channel_properties.py

```diff
@@ -0,0 +1,197 @@
+                max_val=10.0,
+                units=VoltageUnits.VOLTS,
+                bridge_config=BridgeConfiguration.NO_BRIDGE,
+                voltage_excit_source=ExcitationSource.EXTERNAL,
+                voltage_excit_val=0.1,
+                use_excit_for_scaling=False,
+                custom_scale_name="",
+            )
+
+            # Test property initial value.
+            assert not ai_channel.ai_excit_use_for_scaling
+
+            # Test property setter and getter.
+            ai_channel.ai_excit_use_for_scaling = True
+            assert ai_channel.ai_excit_use_for_scaling
+
+            # Test property deleter.
+            del ai_channel.ai_excit_use_for_scaling
+            assert not ai_channel.ai_excit_use_for_scaling
+
+    @pytest.mark.parametrize("seed", [generate_random_seed()])
+    def test_enum_property(self, sim_ts_power_device, seed):
+        """Test for validating channel property of enum type."""
+        # Reset the pseudorandom number generator with seed.
+        random.seed(seed)
+
+        pwr_phys_chan = f"{sim_ts_power_device.name}/power"
+        voltage_setpoint = random.random() * 6.0
+        current_setpoint = random.random() * 3.0
+        output_enable = random.choice([True, False])
+
+        with nidaqmx.Task() as task:
+            pwr_channel = task.ai_channels.add_ai_power_chan(
+                pwr_phys_chan,
+                voltage_setpoint,
+                current_setpoint,
+                output_enable,
+                name_to_assign_to_channel="PowerChannel",
+            )
+
+            # Test property initial value.
+            assert (
+                pwr_channel.pwr_idle_output_behavior
+                == PowerIdleOutputBehavior.MAINTAIN_EXISTING_VALUE
+            )
+
+            # Test property setter and getter.
+            pwr_channel.pwr_idle_output_behavior = PowerIdleOutputBehavior.OUTPUT_DISABLED
+            assert pwr_channel.pwr_idle_output_behavior == PowerIdleOutputBehavior.OUTPUT_DISABLED
+
+            # Test property deleter.
+            del pwr_channel.pwr_idle_output_behavior
+            assert (
+                pwr_channel.pwr_idle_output_behavior
+                == PowerIdleOutputBehavior.MAINTAIN_EXISTING_VALUE
+            )
+
+    @pytest.mark.parametrize("seed", [generate_random_seed()])
+    def test_float_property(self, any_x_series_device, seed):
+        """Test for validating channel property of float type."""
+        # Reset the pseudorandom number generator with seed.
+        random.seed(seed)
+
+        ai_phys_chan = random.choice(any_x_series_device.ai_physical_chans).name
+
+        with nidaqmx.Task() as task:
+            ai_channel = task.ai_channels.add_ai_rtd_chan(
+                ai_phys_chan,
+                name_to_assign_to_channel="RTDChannel",
+                min_val=28.0,
+                max_val=120.0,
+                units=TemperatureUnits.K,
+                rtd_type=RTDType.PT_3750,
```

Comment:
I think the `rtd_type` is the only thing relevant to the `ai_rtd_a` value. defaults for everything else is fine.

Reply:
Added value assignment to `rtd_type` only and removed other assignments